src/Input/Input.jsx

```diff
@@ -140,16 +113,12 @@ export const Input = forwardRef(
             />
           ) : null}
           <Flex pointerEvents='auto' position='absolute' right={2}>
-            {icStateIsNotIconToggle() && currentIcRight ? (
+            {icRight ? (
               <Space mr={1}>
-                <Icon
-                  fontSize={3}
-                  name={currentIcRight}
-                  color={currentIcRightColor}
-                  rotate={currentIcRight === 'loading'}
-                />
+                <Icon fontSize={3} name={icRight} />
               </Space>
-            ) : (
+            ) : null}
+            {type === 'password' && value ? (
```

Comment:
Can't we have disabled password input?

Reply:
I merged with the changes from v1, I did not make myself this change.